kubernetes/ansible/roles/sunbird-monitoring/tasks/main.yml

```diff
@@ -7,15 +7,15 @@
   with_items: "{{ monitoring_stack }}"
 
 - name: Creating sunbird monitoring stack
-  shell: "helm upgrade --install --cleanup-on-fail {{ item }} {{chart_path}}/{{ item }} --namespace monitoring -f /tmp/{{ item }}.yaml"
+  shell: "helm upgrade --install {{ item }} {{chart_path}}/{{ item }} --namespace monitoring -f /tmp/{{ item }}.yaml"
   with_items: "{{ monitoring_stack }}"
 
 - name: Creating sunbird monitoring grafana dashboards
-  shell: "helm upgrade --install --cleanup-on-fail grafana-dashboards {{chart_path}}/dashboards --namespace monitoring"
+  shell: "helm upgrade --install grafana-dashboards {{chart_path}}/dashboards --namespace monitoring"
   tags:
     - dashboards
 
 - name: Install statsd-exporter
-  shell: "helm upgrade --install --cleanup-on-fail statsd-exporter {{chart_path}}/statsd-exporter --namespace {{ namespace }}"
+  shell: "helm upgrade --install statsd-exporter {{chart_path}}/statsd-exporter --set replicaCount={{api_manager_replicacount|d(1)}} --namespace={{ namespace }}"
```

Comment:
Please change this to current implementation. Statsd must be at 1 replica alwyas

Reply:
This is still not updated